### PR TITLE
Help command upgrade (Without .gitignore funky stuff) 

### PR DESCRIPTION
**Pull request by thatrandomperson5**
Same as #76  except no .gitignore funky stuff.

### CODE REVIEW

**Comment by JDJGInc** [comment]
repeated embed thing.

**Comment by JDJGInc** [comment]
Failed :(

File: utils/views.py [hunk 1]
```diff
@@ -528,10 +528,16 @@ async def format_page(self, item):
...
+        super().__init__(title=f"{ctx.bot.user} Help Menu", *args, **kwargs)
+        self.set_author(name=f"Requested by {ctx.author}", icon_url=ctx.author.display_avatar)
+
+
 class SendHelp(Paginator):
     async def format_page(self, item):
-        emby = discord.Embed(description=item, color=15428885)
-        return emby
+        emby = HelpEmbed(self.ctx, description=item, color=15428885)
+        return emby        return emby
```
Comment (JDJGInc) [inline utils/views.py:540]: return emby

**Review by JDJGInc — CHANGES_REQUESTED** [review]
Please change return emby

File: utils/views.py (hunk 1 above)
Comment (thatrandomperson5) [reply utils/views.py]: Done

**Review by JDJGInc — APPROVED** [review]
Looks good to me, I am going to have tom check it out.

**Comment by JDJGInc** [comment]
image

**Comment by thatrandomperson5** [comment]
Yeah, I could change it to help command only if you wish.

**Comment by JDJGInc** [comment]
> Yeah, I could change it to help command only if you wish.

yes please

**Comment by thatrandomperson5** [comment]
👍

**Review by thatrandomperson5 — COMMENTED** [review]
Reviewed for pasting issues, seems to be none.

File: cogs/help.py [hunk 2]
```diff
@@ -7,6 +7,10 @@
 
 
 class JDBotHelp(commands.MinimalHelpCommand):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```
Comment (Soheab) [inline cogs/help.py:11]:
```suggestion
# from typing import Any

    def __init__(self, **options: Any):
        super().__init__(**options)
```

Comment (thatrandomperson5) [reply cogs/help.py:11]: Should I type the whole thing or just this part.

Comment (Soheab) [reply cogs/help.py:11]: that part should be fine for now.

Comment (thatrandomperson5) [reply cogs/help.py]: Done I think

**Pull request merged**